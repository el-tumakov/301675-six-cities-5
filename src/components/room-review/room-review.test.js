import React from "react";
import renderer from "react-test-renderer";
import RoomReview from "./room-review";


const REVIEW = {
  comment: `This villa is perfect in every way: the view on mountains and waterfalls, the hot tub and the villa itself. The evening here became a great continuation of our journeys over country.`,
  date: `2020-10-10T13:38:44.753Z`,
  id: 1,
  rating: 5,
  user: {
    "avatar_url": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/2.jpg`,
    "id": 11,
    "is_pro": false,
    "name": `Jack`
  }
};

it(`Should RoomReview render correctly`, () => {
  const tree = renderer
      .create(
          <RoomReview
            review={REVIEW}
          />
      ).toJSON();

  expect(tree).toMatchSnapshot();
});
