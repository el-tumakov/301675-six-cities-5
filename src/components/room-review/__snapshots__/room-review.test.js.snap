// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RoomReview render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Jack
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "100%",
            }
          }
        >
          5
        </span>
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      This villa is perfect in every way: the view on mountains and waterfalls, the hot tub and the villa itself. The evening here became a great continuation of our journeys over country.
    </p>
    <time
      className="reviews__time"
      dateTime="2020-10-10T13:38:44.753Z"
    >
      October 2020
    </time>
  </div>
</li>
`;
